tests/comfy_cli/command/test_nodes_cli.py

```
@@ -204,3 +204,57 @@ def test_query_flag_rejected(self):
         runner = CliRunner()
         result = runner.invoke(nodes_cmd.app, ["ls", "--query", "produces IMAGE"])
         assert result.exit_code != 0
+
+
+# ---------------------------------------------------------------------------
+# nodes refresh — re-fetch annotation data from comfy-complete
+# ---------------------------------------------------------------------------
+
+
+class TestNodesRefresh:
```

Pylint (4.0.6)

[convention] Missing class docstring

(C0115)